packages/coil-puppeteer-utils/src/lib/logoutCoil.ts

```diff
@@ -9,11 +10,18 @@ export async function logoutCoil(coilPage: Page): Promise<Page> {
   await coilPage.goto(`${COIL_DOMAIN}/settings`)
   await timeout(2e3)
 
-  const menuSelector = `svg[data-cy='hamburger-toggle']`
-  const logoutSelector = "//p[contains(., 'Log out')]"
+  await addCloudFlareAccessHeaders(coilPage)
+
+  const menuSelector = `[data-testid='MenuRoundedIcon']`
+  const signButton = "[data-testid='signout-nav-button']"
 
   try {
     await coilPage.click(menuSelector, { delay: 5e3, clickCount: 1 })
+    const [logoutBtn] = await coilPage.$x(
+      `//div[contains(@class, "MuiListItemButton-root") and contains(., "Sign out")]`
```

Comment:
You can also try something like this:
`'//*[text()="Sign out"]'`

Ideally we can get the team to put in some stable`data-testid` for ALL the relevant elements 
Failing that, the more resilient it is to change the better. 

Some absurd examples to make the point for them selves:
```
/html/body/div/div/div[1]/div[1]/div/ul[3]/div[2]/div[2]/span
```

```
//*[@id="__next"]/div/div[1]/div[1]/div/ul[3]/div[2]/div[2]/span
```

Both the above are quite "fragile"